.gitignore

```diff
@@ -5,3 +5,4 @@ nbproject
 
 .php_cs.cache
 .idea
+.vscode
```

Comment:
It's not related to the project, otherwise we have to remove all IDE possible files.
```suggestion

```